dev-tools/ansible/inventories/standalone/group_vars/pga/vars.yml

```diff
@@ -0,0 +1,70 @@
+#
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+---
+# NOTE: copied from scigap/production/pga_config/seagrid/vars.yml on 8-28-2017. Variables are added and removed all
+# the time so make sure you have the latest by comparing with scigap/production/pga_config/seagrid/vars.yml
+
+# Update all entries below that have CHANGEME as the value
+
+pga_repo: "https://github.com/apache/airavata-php-gateway.git"
+pga_git_branch: "master"
+user: "pga"
+group: "pga"
+doc_root_dir: "/var/www/portals/{{ gateway_id }}"
+#vhost_servername: "CHANGEME"
+#vhost_serveralias: "CHANGEME"
+# Enable and change the following to enable SSL
+#vhost_ssl: True
+#ssl_certificate_file: "/etc/pki/tls/certs/seagrid_org_cert.cer"
+#ssl_certificate_chain_file: "/etc/pki/tls/certs/seagrid_org_interm.cer"
+#ssl_certificate_key_file: "/etc/pki/tls/private/portal.seagrid.key"
+
+
+## Keycloak related variables
+tenant_domain: "{{ gateway_id }}"
+admin_username: "default-admin"
+admin_password: "123456"
+oauth_client_key: "pga"
+oauth_client_secret: "6dd5c5ea-50ce-4a52-8c7c-b11ea80799ac"
+oidc_discovery_url: "https://{{ groups['keycloak'][0] }}/auth/realms/{{ tenant_domain }}/.well-known/openid-configuration"
+
+auth_options:
+  - name: "Airavata"
+    oauth_grant_type: "password"
+oauth_callback_url: "http://{{ vhost_servername }}/callback-url"
+# or if SSL then:
+#oauth_callback_url: "https://{{ vhost_servername }}/callback-url"
+
+gateway_id: "default"
+# relative to document root dir
+experiment_data_dir: "{{ user_data_dir }}/exp-dir"
+gateway_data_store_resource_id: "CHANGEME"
+gateway_data_store_ssh_public_key: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQClwPzT95sK4eA5vuOHe+9cDewjkfBZA0GLt7SfHtJ5Zp/DBWRb5ykobX7JbwJGN9eHmVnW1yL5O708/dLL3y8kT0MjEP5jxZKMSayAiP9c2yN8dved4DIomX3omqsztIcbllei+QM9YvFY7tTtjWFTtAX6WxkZEvzcv2MDTLbgqJ537B4WRRWchXFws98QfDZvlfvdql6BcoZMwDCjdNOl8ZqSTS0PC22+HFesdhMh4mjjNj3Sm2MqxjU8FmSjVlC37dubqS9lZMeKuwsgT1nMjCB8tL+FH5eneNLVKvsvJL4AgQNo84YFi/Gels40zVuCqXVRNsAw/Tn+9sH9JgVx"
+
+## Portal related variables
+super_admin_portal: "true"
+admin_emails: "['dimuthu.upeksha2@gmail.com']"
```

Comment:
@DImuthuUpe  do you want your email address in here? You might end up getting a lot of spam :)

Reply:
My bad. I will remove the email from here